mcos/observation_simulator.py

```diff
@@ -38,3 +41,15 @@ def __init__(self, mu: np.array, cov: np.array, n_observations: int):
     def simulate(self) -> (np.array, np.array):
         x = np.random.multivariate_normal(self.mu.flatten(), self.cov, size=self.n_observations)
         return x.mean(axis=0).reshape(-1, 1), np.cov(x, rowvar=False)
+
+
+def convert_price_history(df: pd.DataFrame):
+    """
+    converts a price history dataframe into expected returns and covariance
+     :param df: Dataframe of price histories indexed by data
+     @return
+    """
+    # Calculate expected returns and sample covariance
+    mu = expected_returns.mean_historical_return(df)
+    cov = risk_models.sample_cov(df)
+    return mu, cov
```

Comment:
Should we move this method should be moved to a `utils.py` file? If I was looking for this method, I wouldn't expect to find it in `observation_simulator.py`.